HelpSource/Classes/Collection.schelp

```diff
@@ -304,10 +304,53 @@ List[1, 2, 3, 3, 4, 3, 4, 3].occurrencesOf(3);
 ::
 
 method::sum
-Answer the sum of the results of strong::function:: evaluated for each item in the receiver. The function is passed two arguments, the item and an integer index.
+Answer the sum of all items or the results of the optional strong::function:: argument evaluated for each item in the receiver. The function is passed two arguments, the item and an integer index.
 code::
 List[1, 2, 3, 4].sum;
+
 (0..8).sum { |i| 1 / (2 ** i) };
+// Only item is used as index is not used in the function.
+::
+discussion::
+If there is no strong::function:: argument, it can be expressed mathematically as follows:
+code::
+[\x_0, \x_1, ..., \x_n].sum // This is conceptual. Do not evaluate.
```

Comment:
instead of "// This is conceptual. Do not evaluate."
// pseudo-code

will be enough.
Commanding the reader is not necessary here, better just inform.

Reply:
And as it's pseudo code, the items probably don't need to be Symbols, but rather just
[x_0, x_1, ..., x_n]
And the comment about the underscored numbers can be removed. I think it's clear what they represent from the latex equation that follows.